samples/cellular/modem_shell/README.rst

```diff
@@ -819,6 +819,26 @@ You can use the ``heap`` command to print kernel and system heap usage statistic
      free:              160
      allocated:          88
 
+----
+
+GPIO pin pulse counter
+======================
+
+You can use the ``gpio_count`` command to count pulses on a given GPIO pin.
+When pulse counting has been enabled, **LED 2** on nRF91 Series DKs shows the state of the pin input.
```

Comment:
It seems there is a limitation that a single GPIO can be counted at a time. This should be mentioned somewhere.